src/plugins/emoji.js

```diff
@@ -1799,6 +1800,7 @@ const AllGithubEmoji = {
   zombie_woman: 'unicode/1f9df-2640',
   zzz: 'unicode/1f4a4',
 };
+/* eslint-enable camelcase */
```

Comment:
Sidenote, we need to make a round of lint fixes and get them all finally done with. The warnings that appear below are annoying, etc.

Reply:
Agreed.